arcgis-ios-sdk-samples/Layers/Display WFS/DisplayWFSViewController.swift

```diff
@@ -0,0 +1,98 @@
+// Copyright 2019 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+class DisplayWFSViewController: UIViewController {
+
+    @IBOutlet private weak var mapView: AGSMapView!
+    private var wfsFeatureTable : AGSWFSFeatureTable!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        // Initialize a map with topographic basemap
+        let map = AGSMap(basemap: .topographic())
+        
+        // A URL to the GetCapabilities endpoint of a WFS service
+        let wfsServiceURL = URL(string: "https://dservices2.arcgis.com/ZQgQTuoyBrtmoGdP/arcgis/services/Seattle_Downtown_Features/WFSServer?service=wfs&request=getcapabilities")!
+        
+        // Name of the table to load
+        let wfsTableName = "Seattle_Downtown_Features:Buildings"
+        
+        // Initialize a WFS feature table with service URL and uniquely identifying table name
+        wfsFeatureTable = AGSWFSFeatureTable(url: wfsServiceURL, tableName: wfsTableName)
+        
+        // Set feature request mode to manual - only manual is supported at v100.5.
+        // In this mode, you must manually populate the table - panning and zooming won't request features automatically.
+        wfsFeatureTable.featureRequestMode = .manualCache
+    
+        // Initialize a feature layer from WFS feature table
+        let wfsFeatureLayer = AGSFeatureLayer(featureTable: self.wfsFeatureTable)
```

Comment:
FYI: Since you created `wfsFeatureTable` in this scope, and therefore know that it is non-`nil`, going through the IUO property creates more work for the compiler. It needs to check for `nil` each time the property is accessed so that it can trap when the value is `none`. In cases like this, it is better to store the needed value to a local variable and use that in the local scope where needed and to update the value of the IUO property. Does that make sense? It's not something you have to fix. Just keep it in mind for the future.

Reply:
> it is better to store the needed value to a local variable and use that in the local scope where needed

Do this in addition to having the IUO property, correct?

Reply:
Correct.